internal/security/proxy/service.go

```diff
@@ -96,23 +95,12 @@ func (s *Service) Init(cert CertificateLoader) error {
 		return err
 	}
 
-	services := config.ListDefaultServices()
-	//TODO: There are specific other services being routed through Kong. This should probably be driven through configuration
-	//      For now, I'm putting this inline
-	services["edgex-support-rulesengine"] = "Rulesengine"
-	services["edgex-device-virtual"] = "Virtualdevice"
-
-	for _, name := range services {
-		params := Configuration.Clients[name]
-		if params.Host == "" {
-			LoggingClient.Warn("missing config information for service: " + name)
-			continue
-		}
+	for _, svc := range Configuration.GetServices() {
 		serviceParams := &KongService{
-			Name:     strings.ToLower(name),
-			Host:     params.Host,
-			Port:     params.Port,
-			Protocol: params.Protocol,
+			Name:     svc.Name,
```

Comment:
The name here needs to be lower case as well so use strings.ToLower(svc.Name) instead.

Reply:
Tingyuz - updated the Name entry with the inclusion of the following:
              Name:     strings.ToLower(clientName),
I have modified the variables to more accurately reflect the usage of the Clients and Name attribute.

Reply:
@tingyuz am I ok to resolve this conflict or do you still need further review on this change?